src/main/java/com/dev/cinema/service/mapper/OrderMapper.java

```diff
@@ -0,0 +1,22 @@
+package com.dev.cinema.service.mapper;
+
+import com.dev.cinema.model.Order;
+import com.dev.cinema.model.Ticket;
+import com.dev.cinema.model.dto.response.OrderResponseDto;
+import java.util.stream.Collectors;
+import org.springframework.stereotype.Service;
+
+@Service
```

Comment:
```suggestion
@Component
```
same for all mappers